cmake/Install.cmake

```diff
@@ -26,23 +26,36 @@ if (UNIX)
   set(CPACK_BINARY_NSIS    OFF CACHE BOOL "Recommended OFF")
 
   if(APPLE)
-    set(CPACK_BINARY_IFW ON CACHE BOOL "Enable to build IFW package, which is the recommended method")
-    set(CPACK_BINARY_STGZ    OFF CACHE BOOL "Recommended OFF")
+    set(CPACK_BINARY_IFW   ON CACHE BOOL "Enable to build IFW package, which is the recommended method")
+    set(CPACK_BINARY_STGZ OFF CACHE BOOL "Recommended OFF")
 
     # Mac Specific options to turn off
     set(CPACK_BINARY_BUNDLE        OFF CACHE BOOL "Recommended OFF")
     set(CPACK_BINARY_DRAGNDROP     OFF CACHE BOOL "Recommended OFF")
     set(CPACK_BINARY_OSXX11        OFF CACHE BOOL "Recommended OFF")
-    set(CPACK_BINARY_PACKAGEMAKER  OFF CACHE BOOL "This was the legacy method on Apple, superseded by IFW.")
+    set(CPACK_BINARY_PACKAGEMAKER  OFF CACHE BOOL "This was the legacy method on Apple, superseded by IFW")
     set(CPACK_BINRARY_PRODUCTBUILD OFF CACHE BOOL "Recommended OFF")
 
   else()
-    set(CPACK_BINARY_IFW     OFF CACHE BOOL "This should be off")
-    set(CPACK_BINARY_STGZ ON CACHE BOOL "Enable to build a Linux sh installer script, which is the recommended method") # Uses STGZ currently (install .sh script CACHE BOOL)
+    # TODO: Make IFW recommended? Deprecate STGZ?
+    set(CPACK_BINARY_IFW   ON CACHE BOOL "Enable to build IFW package, which is the recommended method")
+    set(CPACK_BINARY_STGZ  ON CACHE BOOL "Enable to build a Linux sh installer script, which is the legacy method")
```

Comment:
When this is merged along with #7589, will we automatically get silent install on Linux as well?  Or will that be an additional set of changes later?  If we have a silent option on all three platforms, then this could be marked as legacy, but if not, then there will still be a number of applications where users want to silently install EnergyPlus using the shell script.

Reply:
Yes, it works on Linux as well.